src/test/java/org/breedinginsight/api/v1/controller/ImportControllerIntegrationTest.java

```diff
@@ -156,8 +157,16 @@ public void getSystemMappings() {
         JsonObject result = JsonParser.parseString(response.body()).getAsJsonObject().getAsJsonObject("result");
         JsonArray data = result.getAsJsonArray("data");
 
-        assertEquals(1, data.size(), "Wrong number of results returned.");
-        assertEquals("GermplasmTest", data.get(0).getAsJsonObject().get("name").getAsString(), "Wrong import name returned");
+        boolean importNameFound = false;
```

Comment:
We may want to update this test so it doesn't fail each time a new system mapping is added. It could just look for the test mapping potentially and there could be other tests for each specific system mapping.

Reply:
That is a good point. So for now just remove the line looking for the size of the system mapping list?

Reply:
Ya you could do that. You could also add another test if you wanted using the name query parameter to verify that the germplasm system template exists.

Reply:
Alright, implemented!